fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -651,7 +652,12 @@ protected void updateProcedures() {
             try (Connection c = createConnection()) {
                 try {
                     JdbcTarget target = new JdbcTarget(c);
-                    Db2Adapter adapter = new Db2Adapter(target);
+                    IDatabaseAdapter adapter;
```

Comment:
this is a repeated pattern, I'd suggest adding a single helper method available to all classes.

Reply:
great point! changed to use existing function.